src/test/java/io/reactivex/flowable/FlowableNullTests.java

```diff
@@ -1341,6 +1341,7 @@ public void flatMapNotificationOnErrorNull() {
     }
 
     @Test(expected = NullPointerException.class)
+    @Ignore("No longer crashes with NPE but signals it; tested elsewhere.")
```

Comment:
First it has to get into the 2.x branch for that. Otherwise, there could be a purge of ignored unit test in the future.